ts/TopAppBar/index.tsx

```diff
@@ -0,0 +1,135 @@
+import {MDCTopAppBar} from '@material/top-app-bar';
+import {h} from 'preact';
+import MaterialComponent from '../Base/MaterialComponent';
+
+export interface IRowProps {}
+
+export interface IRowState {}
+
+export class TopAppBarRow extends MaterialComponent<IRowProps, IRowState> {
+  protected componentName = 'top-app-bar__row';
+  protected mdcProps = [];
+
+  protected materialDom() {
+    return <div />;
+  }
+}
+
+export interface ISectionProps extends JSX.HTMLAttributes {
+  'align-start'?: boolean;
+  'align-end'?: boolean;
+}
+
+export interface ISectionState {}
+
+/**
+ * @prop align-end = false
+ * @prop align-start = false
+ * @prop shrink-to-fit = false TODO: Is this sill there?
+ */
+export class TopAppBarSection extends MaterialComponent<
+  ISectionProps,
+  ISectionState
+> {
+  protected componentName = 'top-app-bar__section';
+  protected mdcProps = ['align-start', 'align-end'];
+
+  protected materialDom(props) {
+    return <section {...props}>{props.children}</section>;
+  }
+}
+
+export interface IIconProps extends JSX.HTMLAttributes {
+  navigation?: boolean;
+}
+
+export interface IIconState {}
+
+/**
+ * @prop menu = false TODO: Is this sill there?
```

Comment:
Is there some thing wrong with the interface or with the docs?

Reply:
docs are wrong always